### ISSUE TITLE
Clicking on 'ROS Index' in page header does not take user to domain root

### ISSUE THREAD

**Issue by gavanderhoorn** [rosindex/rosindex.github.io#118]
Instead of going to `http://rosindex.github.io/`, I get `http://rosindex.github.io/packages/` whenever I am on *Home > Packages*, and `http://rosindex.github.io/contribute/` for *Home > Contribute*.

This may just be me having grown used to clicking on site logos and similar to always navigate to the main page, but I thought I'd report it anyway.

**Comment by gavanderhoorn** [comment]
[_includes/header.html](https://github.com/rosindex/rosindex.github.io/blob/4351be00ecdde245b93d11560085304388b78753/_includes/header.html) seems to set the `href` to `site.baseurl`, but looking at the html:
```html
<!-- title -->
<div class="col-xs-12 col-sm-3 col-md-3" style="white-space:nowrap;">
   <a class="site-title" href="">ROS Index</a>
   ...
```

it would appear that is empty? So as `href=""`, the observed behaviour sort-of makes sense.

**Comment by jbohren** [comment]
Yeah, that should just be hard coded to `/`.

**Comment by jbohren** [comment]
Fixed in 6f9b4af4844c5d2dbe8360f50090dc48362a8581 should be live after the next regen.

**Comment by gavanderhoorn** [comment]
Thanks.
